pages/cpm.vue

```diff
@@ -134,6 +103,8 @@ export default {
     },
     zoomToFeature (e) {
       this.map.fitBounds(e.target.getBounds())
+      const layer = e.target
+      this.path_cpm = this.regions.regions[layer._leaflet_id] ? ('cpm_analysis/' + layer._leaflet_id) : ''
     },
     infoDiv (map) {
       this._div = L.DomUtil.create('div', 'info') // create a div with a class "info"
```

Comment:
@Peter9192 In last PR #1 you asked whether it is possible to add this to the main HTML template and simply fetch it (like we do with the mapid div)? I didn't manage to do it since this div is created using the class from leaflet library and attached to the map instance. Any quick idea about it?

Because this info box (div) is created inside the map widget, I cannot control its styling using tailwind css. If we can manage to add it to the main html template and call it there, then we can add css styling using tailwind.

Reply:
So if I understand correctly the L.DomUtil creates an HTML element _inside_ the map - that's why we need it? Nonetheless, you can still add (more) classes to that div, no? If not at once (`L.DomUtil.create('div, 'info bg-blue-400 text-xl')`), then perhaps through `L.DomUtil.addClass(this._div, 'bg-blue-400')`.

Reply:
Corrected!